src/test/java/tests/NavigationMenuTest.java

```diff
@@ -0,0 +1,30 @@
+package tests;
+
+import constants.NavigationLinksConstants;
+import constants.NewsMenuItemsConstants;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class NavigationMenuTest extends BaseTest {
+
+    @DataProvider(name = "menuItemName")
+    public Object[][] menuItemName() {
+        return new Object[][]{{NewsMenuItemsConstants.PEOPLE}, {NewsMenuItemsConstants.CARS}, {NewsMenuItemsConstants.TECHNOLOGIES}, {NewsMenuItemsConstants.REAL_ESTATE}};
+    }
+
+    @Test(dataProvider = "menuItemName")
+    public void openNavigationLink_menuItemsShouldBePresent(String menuItemName) {
```

Comment:
Why do you use different codename conventions for different methods? I think underline should not be used here

Reply:
fixed